tasmota/user_config_override.h

```diff
@@ -4120,113 +4127,164 @@ o888o        o888ooooood8 o88o     o8888o     o888o     o888o o8o        `8
 #define USE_BUTTON_EVENT
 #define SCRIPT_POWER_SECTION
 #define USE_SCRIPT_TIMER
+#define USE_SCRIPT_SUB_COMMAND
+#define USE_SCRIPT_STATUS
+#define USE_WEBSEND_RESPONSE
+#define SCRIPT_STRIP_COMMENTS
+#define USE_ANGLE_FUNC
+#define USE_SCRIPT_FATFS_EXT
+#define USE_SCRIPT_GLOBVARS
+#define SML_REPLACE_VARS
+#define USE_SML_SCRIPT_CMD
+#define USE_SCRIPT_I2C
+#define USE_SCRIPT_SERIAL
+// #define SCRIPT_GET_HTTPS_JP
+#define LARGE_ARRAYS
+#define SCRIPT_LARGE_VNBUFF
+#define USE_GOOGLE_CHARTS
+#define USE_FEXTRACT
+
+#ifdef ESP8266
+  #define USE_SENDMAIL
+
+  #define USE_CHIRP //compilation bug at ESP32
+#endif // ESP8266
+
+#ifdef ESP32
+  #define USE_ESP32MAIL
+  #define MAXVARS 75
+  #define MAXSVARS 15
+  #define MAXFILT 10
+  #define USE_SCRIPT_TASK
+#endif // ESP32
 
 #ifdef ESP32
   #undef OTA_URL
   #define OTA_URL "https://github.com/tasmota/install/raw/main/firmware/unofficial/tasmota32-titanium.bin"
-  #define USE_TLS
-  #define USE_MQTT_TLS
   #define USE_BERRY
   #define USE_BERRY_PSRAM
   #define USE_WEBCLIENT
   #define USE_WEBCLIENT_HTTPS
 #endif  // ESP32
 
-//#ifdef  ESP8266
+// -- Optional modules ----------------------------
 #define ROTARY_V1                                // Add support for Rotary Encoder as used in MI Desk Lamp (+0k8 code)
   #define ROTARY_MAX_STEPS     10                // Rotary step boundary
-//#endif
 
-#define USE_TUYA_MCU                             // Add support for Tuya Serial Dimmer
+
+
+
+
+
+
+
+
+
+
+#define USE_SONOFF_RF                            // Add support for Sonoff Rf Bridge (+3k2 code)
```

Comment:
You include many functions that are not available for esp8266.
Do not remove the `#ifdef ESP8266`here